tests/testing_config.yaml

```diff
@@ -56,7 +56,8 @@ training:
 
 predict:
     decals_frame: ${paths.decals}/tractor-3366m010.fits
-    weight_save_path: ${paths.pretrained_models}/sdss.pt
+    weight_save_path: ${paths.pretrained_models}/r_long.pt
```

Comment:
Did you also intend to add this model to the repo? If not, we should probably keep the path to `sdss.pt`

Reply:
This is temporary so one of the test cases can be debugged on my end (`test_simulate.py`) until I create some new example tile catalogs.